Source/BluemixAppID/internal/AppIDConstants.swift

```diff
@@ -178,6 +178,7 @@ internal class AppIDConstants {
     internal static let USER_IDENTITY_LABEL = "userIdentity"
     // labels
     
+    internal static let AnonymousIdpName = "appid_anon"
     internal static let BMSSecurityErrorDomain = "com.ibm.mobilefirstplatform.clientsdk.swift.bmssecurity"
```

Comment:
I don't know about this. It is not related to the profiles API, and I am sure there are more of those hiding. Let's finish this, and then we'll do sweep around the code and remove any such "old" constants